elements/lisk-transactions/test/15_pom_transaction.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+describe('Proof-of-misbehavior transaction', () => {
+	describe('validateAsset', () => {
+		it.todo(
+			'when header1.height is greater than or equal to header2.height but equal maxHeighPrevoted it should not return errors',
+		);
+
+		it.todo(
+			'when header1.height is greater than header2.maxHeightPreviouslyForged it should not return errors',
+		);
+
+		it.todo(
+			'when header1.maxHeightPrevoted is greater than header2.maxHeightPrevoted it should not return errors',
+		);
```

Comment:
the function checking for contradiction will reorder the headers (according to the rules defined in said function). This order does not have to be respected when the POM is submitted.

What I mean: header1 in the POM might have to be b2 in the bft contradicting function. 

If I can suggest something: implement the BFT contradicting function as in the BFT LIP and call this function. This should already exist as it is used else where in BFT.

Reply:
I will reword it to be general: if one is greater than the other.